src/tests/Myr.test.js

```diff
@@ -853,4 +853,19 @@ describe("Other Myr functionality", () => {
         expect(response).toEqual({ "test1": 1, "test2": 3 });
     });
 
+    it("Should accept any caps or lowercase letters", () => {
```

Comment:
You should do some more testing with other colors besides just red.
Red is the default color for MYR, so try some other colors before testing red.